internal/reaper/reaper.go

```diff
@@ -35,14 +36,30 @@ type Reaper struct {
 
 func NewReaper(ctx context.Context, config *Config, notifier notification.Notifier) (*Reaper, error) {
 	logger := log.Ctx(ctx)
-	k8s, err := kubernetes.ConnectToKubernetes()
+
+	cfg, err := rest.InClusterConfig()
```

Comment:
Hey, I'm remember that code. ;-)

Reply:
Yes, somehow the dep didn't work as expected, so I had to write it mysefl.